README.md

```diff
@@ -5,6 +5,20 @@
 |                                            |                                            |
 | ------------------------------------------ | ------------------------------------------ |
 | ![Screenshot 1](assets/landscape_dark.png) | ![Screenshot 2](assets/portrait_light.png) |
+# you need to follow the below steps to use this template and to render the output
```

Comment:
## Build the project locally
You need to follow the below steps to use this template and render the output: